lib/filter.g

```diff
@@ -171,6 +171,60 @@ BIND_GLOBAL( "InstallTrueMethodNewFilter", function ( tofilt, from )
 end );
 
 
+#############################################################################
+##
+#F SuspendMethodReordering()
+#F ResumeMethodReordering()
+#F ResetMethodReordering()
+##
+##  <#GAPDoc Label="MethodReordering">
+##  <ManSection>
+##  <Func Name="SuspendMethodReordering" Arg=""/>
+##  <Func Name="ResumeMethodReordering" Arg=""/>
+##  <Func Name="ResetMethodReordering" Arg=""/>
+##
+##  <Description> These functions control whether the method reordering process
```

Comment:
Virtually every other GAPDoc comment in the library has `<Description>` and `</Description>` on their own lines. Just sayin'